quote.cpp

```diff
@@ -0,0 +1,44 @@
+#include <string>
+
+namespace Sass {
+  using namespace std;
+
+  string unquote(const string& s)
+  {
+    if (s.empty()) return "";
+    if (s.length() == 1) {
+      if (s[0] == '"' || s[0] == '\'') return "";
+    }
+    char q;
+    if      (*s.begin() == '"'  && *s.rbegin() == '"')  q = '"';
+    else if (*s.begin() == '\'' && *s.rbegin() == '\'') q = '\'';
```

Comment:
Thanks. I missed this, because I did not re-inspect `unquote` after I handled the merge from #779 , commit https://github.com/sass/libsass/commit/5483186ab9406fa69bfcc48b59b2a76009d9483d.